src/components/Resources/index.js

```diff
@@ -5,23 +5,44 @@ import PersonalMenu from '../PersonalMenu';
 import Search from '../Search';
 import Grid from '@material-ui/core/Grid';
 import { ResourceCard } from './ResourceCard';
+import * as helper from '../helper';
 
 function Resources({ getResourcesUrl }) {
   const [resources, setResources] = useState([]);
+  const [searchValue, setSearchValue] = useState('');
+  const [loading, setLoading] = useState(true);
+  const [errorMessage, setErrorMessage] = useState(null);
 
   useEffect(() => {
     axios
       .get(getResourcesUrl)
       .then(function(response) {
         // handle success
-        setResources(response.data.results);
+        setResources(response.data);
+        setLoading(false);
       })
       .catch(function(error) {
         // handle error
         console.log(error);
       });
   }, []);
 
+  const search = searchValue => {
```

Comment:
I think the search function can be refactored into it's own file.

Reply:
Agree, although... tempted to finish up in a future refactor. I logged a TODO here. :)